packages/web/src/pages/settings-page/components/desktop/DeveloperApps/CreateNewAppPage.tsx

```diff
@@ -84,7 +86,8 @@ export const CreateNewAppPage = (props: CreateNewAppPageProps) => {
   const initialValues: DeveloperAppValues = {
     userId,
     name: '',
-    description: ''
+    description: '',
+    imageUrl: undefined
```

Comment:
this was a subtle / annoying bug that I thought to be explicit about. Let me add a comment. If it's `''` it will be sent as empty string and fail fqdn check. description being `''` is probably wrong actually.